pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
```

Comment:
Agreed but I think this can be said more concisely.  Perhaps "Download a file saving it under the same name as indicated in the url"?